django/middleware/security.py

```diff
@@ -52,4 +53,13 @@ def process_response(self, request, response):
                 if isinstance(self.referrer_policy, str) else self.referrer_policy
             ))
 
+        if self.feature_policy:
+            response.setdefault('Feature-Policy', '; '.join(
```

Comment:
It will be better to use an `in` check, so that the entire construction can be skipped if not needed.

Reply:
Are you sure you want to skip if the header already exists? Another option is to append it (by setting the header twice -- don't know if Django/WSGI supports that -- or manually).

Reply:
The question is whether duplicate definitions would stack - I think a duplicate would override the one already set - and also what the priority ought to be if it did stack. Because this is in the middleware, it would be applied *after* the header explicitly set on the response which I think would be unexpected.

This check is to allow explicit overrides in views and is the same behaviour used for `Referrer-Policy` and other headers in this middleware.